components/calendar/src/any_calendar.rs

```diff
@@ -609,20 +609,25 @@ impl AnyCalendarKind {
     ///
     /// Returns None if the calendar is unknown
     pub fn from_bcp47_string(x: &str) -> Option<Self> {
+        Self::from_bcp47_bytes(x.as_bytes())
+    }
+    /// Construct from a BCP-47 byte string
```

Comment:
I think our plan is to expect `std::string_view` is always UTF8 (though sometimes allowed to be ill-formed).

I don't want to doc the encoding that way because in JS there's no choice but using UTF16.